scipy/stats/stats.py

```diff
@@ -2516,6 +2517,116 @@ def iqr(x, axis=None, rng=(25, 75), scale='raw', nan_policy='propagate',
     return out
 
 
+
+def mad(x, axis=None, nan_policy='propagate', keepdims=False):
```

Comment:
- I don't think that there are other stats functions with `keepdims` except `iqr`. I don't think it's needed.
- `axis=0` is normally the default for `scipy.stats`. `iqr` is again the only exception.
- both statsmodels and R have the normalization constant and `center` as keywords. I think they should also be added here.
- I'm not so sure about the `low`/`high` keywords that [R has](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/mad.html). opinions?

Reply:
Alright, cool. I'll add the normalisation constant and the center keywords. I don't think the `low/high` keywords are needed, and they will probably require us to re-implement the median function, since `np.median` can't do this as yet.